src/commands/job/execute.js

```diff
@@ -15,6 +15,7 @@ import path from 'path'
  * @param {boolean} output - flag indicating if CLI should print out job output. If string was provided, it will be treated as file path to store output.
  * @param {boolean | string} output - flag indicating if CLI should print out job output. If string was provided, it will be treated as file path to store output. If filepath wasn't provided, output.json file will be created in current folder.
  * @param {boolean | string} log - flag indicating if CLI should fetch and save log to the local folder. If filepath wasn't provided, {job}.log file will be created in current folder.
```

Comment:
This description should be updated (and maybe the parameter name)

For executing jobs, we always provide a `logFile` (path to a file), never a folder